src/shared/components/MapTo.tsx

```diff
@@ -226,7 +226,14 @@ export const MapToDropdownBasic = ({
   children,
   config,
 }: MapToDropdownBasic) => (
-  <Dropdown visibleElement={<Button variant="tertiary">{children}</Button>}>
+  <Dropdown
+    // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
Not sure how different this is from the function as child pattern, but here it gets flagged but the linting rule. I've moved them outside the component where possible (when the button was static), but when the content depend on props or other variable (like here, it depends on the MapTo children) I've kept it inside and disabled the rule